release_notes/addtl-release-notes.adoc

```diff
@@ -65,7 +65,7 @@ xref:../observability/distr_tracing/distr-tracing-rn.adoc#distr-tracing-rn[{DTPr
 {nbsp} +
 link:https://docs.redhat.com/en/documentation/red_hat_openshift_gitops/#Get%20started[{gitops-title}] +
 {nbsp} +
-link:https://docs.redhat.com/en/documentation/red_hat_openshift_local/#Release%20Notes[{openshift-local-productname}] +
+link:https://crc.dev/docs/introducing/#about[{openshift-local-productname} (Upstream CRC documentation)] +
```

Comment:
Do we need to state `CodeReady Containers (CRC)` here? If not, feel free to ignore my comment.

Reply:
Thanks for the comment. A writer that knows about OCP Local writer is due to review the content also. I'll ask them to confirm.

Reply:
CRC does not mean Codeready Containers anymore, it turned into a recursive acronym: CRC Runs Containers. Red Hat OpenShift Local is the distribution of the `crc` CLI by Red Hat that includes a fully functional OpenShift Container Platform single node bundle.

The `#about` anchor might be unnecessary.

You could also point to the docs root page https://crc.dev/docs and let the redirections happen.